src/main/resources/project/jython/update_app.jython

```diff
@@ -14,25 +14,41 @@
 #  limitations under the License.
 #
 
+[% preamble %]
+
+print('Going for updating application')
+connect(r'[% wl_username %]', r'[% wl_password %]', r'[% admin_url %]')
+applicationName = r'[% app_name %]'
+planPath = r'''[% plan_path %]'''.strip()
+options = r'[% options %]'
+applicationPath = r'''[% app_path %]'''.strip()
```

Comment:
Please, see how r'''$[param]''' is implemented in websphere:
```
r'''
$[param]
'''.strip()
```
This form allows us to have trailing backslash.